src/avif.c

```diff
@@ -820,9 +827,10 @@ avifBool avifCleanApertureBoxConvertCropRect(avifCleanApertureBox * clap,
                                              avifPixelFormat yuvFormat,
                                              avifDiagnostics * diag)
 {
+    (void)yuvFormat;
```

Comment:
If we want to preserve the old behavior, we cannot ignore `yuvFormat`.

It would be useful to move lines 804-806 to a new function that takes `const avifCropRect * cropRect` and `avifPixelFormat yuvFormat` as input and returns the `upsampleBeforeCropping` boolean. Then we can call that new function here (to preserve the old behavior) and at line 804.

Reply:
Done.